response_operations_ui/views/collection_exercise.py

```diff
@@ -216,3 +217,28 @@ def _format_ci_file_name(collection_instruments, survey_details):
 def _get_form_type(file_name):
     file_name = file_name.split(".")[0]
     return file_name.split("_")[2]  # file name format is surveyId_period_formType
+
+
+@collection_exercise_bp.route('/<short_name>/<period>/edit-collection-exercise-details', methods=['GET'])
+@login_required
+def view_collection_exercise_details(short_name, period):
+    ce_details = collection_exercise_controllers.get_collection_exercise(short_name, period)
+
+    form = EditCollectionExerciseDetailsForm(form=request.form)
+
+    return render_template('edit-collection-exercise-details.html', survey_ref=ce_details['survey']['surveyRef'],
+                           form=form, short_name=short_name, period=period,
+                           user_description=ce_details['collection_exercise']['userDescription'],
+                           collection_exercise_id=ce_details['collection_exercise']['id'])
+
+
+@collection_exercise_bp.route('/<short_name>/<period>/edit-collection-exercise-details', methods=['POST'])
+@login_required
+def edit_collection_exercise_details(short_name, period):
+    form = request.form
+
+    collection_exercise_controllers.update_collection_exercise_details(form.get('collection_exercise_id'),
+                                                                       form.get('user_description'),
+                                                                       form.get('period'))
+
+    return redirect(url_for('surveys_bp.view_survey', short_name=short_name, period=period))
```

Comment:
Is `period=period` needed here for [view_survey](https://github.com/ONSdigital/response-operations-ui/blob/master/response_operations_ui/views/surveys.py#L26)?